application/api/meetings.go

```diff
@@ -0,0 +1,8 @@
+package api
+
+// MeetingName just gives the name of a meeting a/k/a Event, to serve as a focal point for finding, answering, carrying, and exchanging requests
+// swagger:model
+type MeetingName struct {
```

Comment:
I think we could use a full `Meeting` struct even though the UI only uses the name of the meeting when it's a child object of `Watch`. The `LoadMeeting` function loads all of the non-struct fields anyway. Might as well use them. Otherwise, if performance is a concern, we could get just the meeting name from the database.

Reply:
I would rather not include the meeting's related fields.  Is that OK?

Reply:
Do you mean not to include struct fields on Meeting (e.g. File, Location) or not to include any fields except the name?